packages/api/test/unit/core/file.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { expect } from 'chai';
 import { deserialize, serialize } from 'surrial';
 
-import { File } from '../../../core';
+import { File } from '../../../src/core';
```

Comment:
I don't quite like pathing to src from tests / packages

Reply:
Ah, you mean just import from `'../../../core'`? Yes that would be possible here because of the new `d.ts` files.

Reply:
sorry, that doesn't work. See my latest comment